code/index.html

```diff
@@ -5,32 +5,73 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="./image/favicon.png"
+      alt="favicon"
+    />
+    <title>Cluedo</title>
   </head>
+
+  <!-- LOGO CONTAINER -->
   <body>
+    <section class="top-container">
```

Comment:
could perhaps name the class "logo-container" to follow previous info?

Reply:
Good naming suggestion